features/steps/common_http.py

```diff
@@ -0,0 +1,110 @@
+# Copyright © 2023, José Luis Segura Lucas, Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Common steps for HTTP related operations."""
+
+import json
+
+import jsonschema
+import requests
+from behave import given, then, when
+
+
+@when("I request the {endpoint} endpoint in {url} with {body} in the body")
+def request_endpoint_with_body(context, endpoint, url, body):
+    """Perform a request to the local server with a given body in the request."""
+    context.response = requests.get(
+        f"http://{url}/{endpoint}",
+        data=body,
+    )
```

Comment:
port might be configured too, right?

Reply:
The URL, in this case, includes both hostname:port. Do you think it could be better to separate them as 2 arguments?

Reply:
probably, but as you wish

Reply:
Done!